projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js

```diff
@@ -40,6 +41,11 @@ const updateLaunchpadSaveModalBrowserConfig = config => {
 
 export const settings = {
 	render: function LaunchpadSaveModal() {
+		const [ experimentVariationName, setExperimentVariationName ] = useState();
+		const sessionVariationName = window.sessionStorage.getItem(
```

Comment:
Just to be clear we're basically using the `window.storageSession` to decide treatment/control, locally, that's alright as long as this is done in the debug/dev mode we don't want to exposed this and allow user to arbitrarily tweak their assigned experienced. 

Is there a reason you chose this approach instead of the assignment API ?